demo/summer-2016.html

```diff
@@ -219,9 +219,10 @@
     var session = editor.getSession();
     session.setMode("ace/mode/papeeria_latex");
     session.setUseWorker(true);
-    window.onload = function() {
+
+    var initSpellCheck = function() {
       session._emit("changeSpellingCheckSettings", {
-        alphabet: 'abcdefghijklmnopqrstuvwxyz',
+        alphabet: 'abcdefghijklmnopqrstuvwxyz1234567890',
```

Comment:
Before I added digits to the alphabet, numbers weren't highlighted at all. I barely figured it out that I should complement the alphabet to make it work.